### PR TITLE
missing subscript

### PR DESCRIPTION
**Pull request by niangaotuantuan**
D_val for validation set

### CODE REVIEW

File: lecture_note.tex [hunk 1]
```diff
@@ -881,7 +881,7 @@ \subsection{Early Stopping}
 and the validation cost as
 \begin{align}
     \label{eq:val_c}
-    C_{\val}(\TT) = \frac{1}{|D_{\val}|} \sum_{(x,y) \in D_{\val}} D(\hat{y}, y).
+    C_{\val}(\TT) = \frac{1}{|D_{\val}|} \sum_{(x,y) \in D_{\val}} D_{\var}(\hat{y}, y).
```
Comment (kyunghyuncho) [inline lecture_note.tex]: In fact, this should stay same, but the second `D_{\train}` in the line 879 should be changed to `D`

Comment (niangaotuantuan) [inline lecture_note.tex]: :) I got it. But in this way, maybe a better definition of D is more proper?

Comment (kyunghyuncho) [inline lecture_note.tex]: Yes, that'll be better! How about you modify this PR to change the line 879 first? I'll merge it. Later either you can make a PR for the definition of `D`, but I can do it myself.

Comment (niangaotuantuan) [inline lecture_note.tex]: :) Done.

**Comment by kyunghyuncho** [comment]
Thanks!

**Pull request merged**